application/cli-app/src/main/java/org/mobilitydata/gtfsvalidator/Main.java

```diff
@@ -67,6 +65,7 @@ public static void main(String[] args) {
                 final ValidateGtfsTypes validateGtfsTypes = config.validateGtfsTypes();
                 final ProcessParsedAgency processParsedAgency = config.processParsedAgency();
                 final ProcessParsedRoute processParsedRoute = config.processParsedRoute();
+                final GetUsecaseCollectionForFile getUsecaseCollectionForFile = config.getValidationUsecaseAll();
```

Comment:
As discussed, rename `GetUseCaseCollectionForFile` to `GetFileSpecificUseCaseCollection`